noxfile.py

```diff
@@ -38,32 +38,11 @@ def import_package(session):
 
 
 @nox.session
-def build_docs(session):
-    session.install(".[dev,docs]")
-    session.run(
-        "sphinx-build",
-        *sphinx_opts,
-        *session.posargs,
-    )
-
-
-@nox.session
-def build_docs_nitpicky(session):
+def docs(session):
```

Comment:
We had two doc build options, which were with or without the "nitpicky" option of `sphinx-build` (which turns warnings about broken references into errors). I don't think we have a need for building docs without the "nitpicky" option turned on, so I renamed `build_docs_nitpicky` to a much more concise `docs`.